freetextresponse/freetextresponse.py

```diff
@@ -26,16 +25,14 @@
 
 
 @XBlock.needs("i18n")
-class FreeTextResponse(
-    EnforceDueDates,
-    MissingDataFetcherMixin,
-    StudioEditableXBlockMixin,
-    XBlock,
-):
+class FreeTextResponse(EnforceDueDates, MissingDataFetcherMixin, StudioEditableXBlockMixin, XBlock):
```

Comment:
please keep the arguments in the multiline format (especially with the trailing comma at the end), it makes diffs cleaner and reduces merge conflicts...

Reply:
Specifically here, it doesn't look like anything changed except reformatting the lines?
Especially in that case, we should not make this change as it's not related to the issue at hand.

Reply:
@caesar2164 @stvstnfrd I've used `render_django_template` which is more XBlock friendly, especially for `i18n`.